Packs/CommonDashboards/ReleaseNotes/1_5_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### New: UseCaseAdoptionMetrics
+
```

Comment:
Add release note comment regarding your change.
For examples view:
https://xsoar.pan.dev/docs/documentation/release-notes